SU-SWS/stanford_paragraph_types
# Earth 117 postcard type

# READY FOR REVIEW

# Summary
- Created postcard paragraph type and use postcard pattern

# Needed By (Date)
- 6/15/17

# Urgency
- low urgency

# Steps to Test
0. Refresh local environment (optional)
1. Checkout branch
2. enable module `drush en stanford_paragraph_postcard`
3. Create a new complex page `node/add/complex_page`
4. Add a `Postcard` paragraph item and populate as desired (if it does not display, configure the field in complex_page)
5. save the node.
6. Verify pattern `Postcard` is used - themeing will come later (https://stanfordits.atlassian.net/browse/EARTH-129)
